README.md

```diff
@@ -148,7 +148,7 @@ This method describes required parameters needed to retrieve an access token, us
     + parameters :
         * `token_endpoint`: the URL to which send the refresh token
         * `client_id` : the client ID
-        * `rtoken_var_name`: for practical reasons, the refresh token is provided using environment variables. This entry describes the name of the variable containing the secret refresh token
+        * `rtoken_from_var`: for practical reasons, the refresh token is provided using environment variables. This entry describes the name of the variable containing the secret refresh token
```

Comment:
This config isn't valid anymore. It got migrated in the 3 to 4 update, so updated to use the right version